### DISCUSSION TITLE
How can I use Zenoh-router on an operating system that does not support Docker/Rust?

### DISCUSSION THREAD

**Discussion by yuyuyuyi2001** [eclipse-zenoh/roadmap#175] [Q&A]
I want to implement Zenoh Pico on sylixOS through cross compilation, but currently the compiled result only has one library. Is this fully functional and correct? 
![1735051286762](https://github.com/user-attachments/assets/5e82a399-2296-4f41-b4f3-4534ee9e08d2)

Also, I would like to conduct some example validations, but SylixOS does not support Rust and Docker. Is there any other way to run Zenoh router? Thank you!

**Comment by sashacmc** [comment]
Hello @yuyuyuyi2001, it is impossible to build a Zenoh Router without Rust. But you can use Zenoh-Pico in peer/multicast mode
https://github.com/eclipse-zenoh/zenoh-pico?tab=readme-ov-file#34-basic-pubsub-example---p2p-over-udp-multicast
